src/seedu/addressbook/AddressBook.java

```diff
@@ -486,12 +486,38 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
             final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
+            
+            convertToUppCase(keywords);
+            convertToUppCase(wordsInName);
+            
             if (!Collections.disjoint(wordsInName, keywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
+    
+    
+    /**
+     * Convert names to upper case for comparison later in getPersonsWithNameContainingAnyKeyword
+     * method so that find command will be case insensitive
+     * 
+     * @param names to be changed to uppercase
+     * 
+     */
+    
+    private static void convertToUppCase(Collection<String> names) {
+        ArrayList<String> namesInCaps = new ArrayList<>();
+        for (String name : names) {
+            String capName = name.toUpperCase();
+            namesInCaps.add(capName);
+        }
+        names.clear();
```

Comment:
in the current form it is violating SLAP, 'cos you are converting to upper case and you are updating the input separately.

also, consider using some Java string library functions to do case insensitive operations